### PR TITLE
CB-437: Add entity metadata to review get endpoints

### PR DESCRIPTION
**Pull request by anshg1214**
* Return average ratings for entity if the entity_id is given
* Add parameter to include the metadata for the entities

### CODE REVIEW

**Review by alastair — CHANGES_REQUESTED** [review]

File: critiquebrainz/ws/review/views.py [hunk 1]
```diff
@@ -389,9 +398,10 @@ def review_list_handler():
                               f'offset={offset}', f'language={language}', f'review_type={review_type}')
     cached_result = cache.get(cache_key, REVIEW_CACHE_NAMESPACE)
 
-    if cached_result:
+    if False:
```
[the remark this answers is not in the document]
Comment (anshg1214) [reply critiquebrainz/ws/review/views.py]: I've already fixed this in the previous commit :)

File: critiquebrainz/ws/review/views.py [hunk 2]
```diff
@@ -375,6 +378,12 @@ def review_list_handler():
     if not sort_order:
         sort_order = 'desc'
 
+    # If an entity_id is given, then also include the average rating for the entity.
+    if entity_id:
```
[the remark this answers is not in the document]
Comment (anshg1214) [reply critiquebrainz/ws/review/views.py:384]: Done 👍

File: critiquebrainz/ws/review/views.py [hunk 3]
```diff
@@ -422,6 +458,9 @@ def review_list_handler():
                    expirein=REVIEW_CACHE_TIMEOUT,
                    namespace=REVIEW_CACHE_NAMESPACE)
 
+    if include_avg_rating:
+        return jsonify(limit=limit, offset=offset, count=count, reviews=reviews, avg_rating=avg_rating)
```
[the remark this answers is not in the document]
Comment (anshg1214) [reply critiquebrainz/ws/review/views.py]: Done 👍

**Review by alastair — APPROVED** [review]
looks good now, thanks for fixing the last few issues!

**Pull request merged**